exporter/collector/metrics.go

```diff
@@ -657,7 +657,7 @@ func (m *metricMapper) histogramToTimeSeries(
 	point pmetric.HistogramDataPoint,
 	projectID string,
 ) []*monitoringpb.TimeSeries {
-	if point.Flags().NoRecordedValue() || !point.HasSum() {
+	if point.Flags().NoRecordedValue() || !point.HasSum() || point.ExplicitBounds().Len() == 0 {
```

Comment:
this change affected the `_expected` test fixture:
```diff
diff --git a/exporter/collector/integrationtest/testdata/fixtures/metrics/prometheus_empty_buckets_expected.json b/exporter/collector/integrationtest/testdata/fixtures/metrics/prometheus_empty_buckets_expected.json
index 6633318..96c3f83 100644
--- a/exporter/collector/integrationtest/testdata/fixtures/metrics/prometheus_empty_buckets_expected.json
+++ b/exporter/collector/integrationtest/testdata/fixtures/metrics/prometheus_empty_buckets_expected.json
@@ -3,44 +3,6 @@
     {
       "name": "projects/fakeprojectid",
       "timeSeries": [
-        {
-          "metric": {
-            "type": "workload.googleapis.com/kv_prober_write_latency",
-            "labels": {
-              "service_instance_id": "0.0.0.0:2112",
-              "service_name": "otel-collector"
-            }
-          },
-          "resource": {
-            "type": "generic_task",
-            "labels": {
-              "job": "otel-collector",
-              "location": "global",
-              "namespace": "",
-              "task_id": "0.0.0.0:2112"
-            }
-          },
-          "metricKind": "CUMULATIVE",
-          "valueType": "DISTRIBUTION",
-          "points": [
-            {
-              "interval": {
-                "endTime": "1970-01-01T00:00:00Z",
-                "startTime": "1970-01-01T00:00:00Z"
-              },
-              "value": {
-                "distributionValue": {
-                  "bucketOptions": {
-                    "explicitBuckets": {}
-                  },
-                  "bucketCounts": [
-                    "0"
-                  ]
-                }
-              }
-            }
-          ]
-        },
```

Reply:
Thats interesting.  How were integration tests passing before if there was an invalid histogram?

Reply:
Sorry, this wasn't a test that existed before. I added it, and did a before and after with the logic change to see that the (new) fixture was updated.